content/building/encrypting.md

```diff
@@ -22,11 +22,17 @@ The encrypted value will look something like this:
 Encrypted(Z0FBQUFBQmRyY1FLWXIwVEhqdWphdjRhQ0xubkdoOGJ2bThkNmh4YmdXbFB3S2wyNTN2OERoV3c0YWU0OVBERG42d3Rfc2N0blNDX3FfblZxbUc4d2pWUHJBSVppbXNXNC04U1VqcGlnajZ2VnJVMVFWc3lZZ289)
 ```
 
-Note that when you upload a **file** for encryption, it is encoded to `base64` and would have to be decoded during the build. If you want to encrypt a file (plaintext, JSON, plist), copy-paste the contents of it to the encryption interface and use the encrypted value in your configuration.
+If you wish to encrypt a **file** to add to your workflow, you will first have to base64 encode it, and then encrypt the received string. To use the file you will have to decode it during the build.
```

Comment:
```suggestion
If you wish to encrypt a **file** to add to your workflow, you will first have to base64-encode it and then encrypt the received string. To use the file you will have to decode it during the build.
```

Reply:
Is it important to clarify at what stage the decoding should happen in the build? That is where in the yaml they should have the line to decode the file?

Reply:
Ok, I saw you mention later in the doc that it should be done in scripts.